core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -1001,6 +1001,30 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
     assertTrue(assertThrows(classOf[ExecutionException], () => describeResult2.values.get(invalidTopic).get).getCause.isInstanceOf[InvalidTopicException])
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = Array("zk", "kraft"))
+  def testIncludeDocumentation(quorum: String): Unit = {
+    val consumer = createConsumer()
+    subscribeAndWaitForAssignment(topic, consumer)
+    client = createAdminClient
+
+    val resource = new ConfigResource(ConfigResource.Type.TOPIC, topic)
+    val includeDescribe = new DescribeConfigsOptions().includeDocumentation(true)
+    var describeConfigs = client.describeConfigs(Collections.singletonList(resource), includeDescribe)
+    val pattern = """documentation=([^,]*?)\)""".r
```

Comment:
How about `documentation=(.*\)?)`

Reply:
IMHO, it's more precise and robust as it avoids matching commas, ensuring the `documentation` value is accurately captured up to the first comma or closing.
But it will be more complicated than `[^,]`